source/nodejs/adaptivecards-designer/src/adaptive-card-schema.ts

```diff
@@ -147,6 +147,7 @@ export const adaptiveCardSchema =
           "$ref": "#/definitions/Actions"
```

Comment:
1. We should be restricted to only one carousel.
2. After I click 'Preview' button, then all the changes are gone after I added 2 or more carousel.

![image](https://user-images.githubusercontent.com/6290692/179059847-aef69295-4726-40c5-9ba2-15603ce5cb77.png)

![image](https://user-images.githubusercontent.com/6290692/179060029-dc59c831-b5b5-46f1-966e-a8a7703a1891.png)

Reply:
I'll look into this!

Reply:
Can we prevent adding carousel within a carousel? that might be easier for us to maintain and also prevent the user from creating a schema to a bad experience...

Reply:
Would we also want to prevent multiple carousels within one card (assuming we remove some of the singleton behavior)